src/services/user.services.js

```diff
@@ -0,0 +1,30 @@
+export const userService = {
+    login
+};
+
+function login(email, password) {
+    const requestOptions = {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ email, password })
+    };
+
+    return fetch("/api/v1/rest-auth/login/", requestOptions)
```

Comment:
You can create a network layer using your http client(fetch in your case) which handles all the API calls stuff(add headers, handles error) and consume it from your services, so that you will save from repeat a lot of code. there is a good example in our base if you wanna take a look.

Reply:
Thanks for the suggestions @manema, I'll take a look of them and try to add that in new PRs.